runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1805,13 +1805,26 @@ Java_java_lang_Class_getNestHostImpl(JNIEnv *env, jobject recv)
 	J9Class *nestHost = clazz->nestHost;
 
 	if (NULL == nestHost) {
-		if (J9_VISIBILITY_ALLOWED == vmFuncs->loadAndVerifyNestHost(currentThread, clazz, J9_LOOK_NO_THROW)) {
-			nestHost = clazz->nestHost;
+		J9Class *clazzToUse = clazz;
+		J9JavaVM *vm = currentThread->javaVM;
```

Comment:
This local could be pushed before, and reused in, the declaration of `vmFuncs`.

Reply:
Changed the code to use `#if JAVA_SPEC_VERSION >= 15`. There is no need to have this local variable `vm` anymore.